doc/source/dev/development_environment.rst

```diff
@@ -339,9 +278,10 @@ needs a ``.gdbinit`` file with the following contents:
 
     add-auto-load-safe-path ~/.pyenv
 
-Instead of plain ``gdb`` you can of course use your favourite
-alternative debugger; run it on the python binary with arguments
-``runtests.py -g --python mytest.py``.
+.. 
+    Instead of plain ``gdb`` you can of course use your favourite
+    alternative debugger; run it on the python binary with arguments
+    ``runtests.py -g --python mytest.py``.
```

Comment:
This probably needs upstream support in `spin` to work nicely with e.g. `lldb`, otherwise it probably belongs in the advanced debugging section.

Reply:
agreed, done